terminus/builders/procedural_city_builder.py

```diff
@@ -16,50 +16,64 @@
 
 import pickle
 
+import os
+import sys
+import subprocess
+
+import logging
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
 
 class ProceduralCityBuilder(object):
-    def __init__(self, verticesFilename, polygonsFilename):
+    def __init__(self, verticesFilename=None, polygonsFilename=None, size=1500):
         self.verticesFilename = verticesFilename
         self.polygonsFilename = polygonsFilename
+        self.ratio = 100  # ratio between pcg and gazebo (1unit = 100m)
+        self.size = int(size)
 
     def get_city(self):
         city = City()
-        ratio = 100
+
+        # If no verticesFilename or polygonsFilename is specified,
+        # generate new ones using procedural_city_generation
+        if self.verticesFilename is None or self.polygonsFilename is None:
```

Comment:
Just to double-check: what happens if you provide `verticesFilename` but not `polygonsFilename` (or vice-versa)?